index.html

```diff
@@ -52,11 +52,241 @@ <h2>I’am a Software Developer</h2>
+                                    <li class="bootstrap"><span>bootstrap</span></li>
+                                    <li class="ruby"><span>Ruby</span></li>
+                                </ul>
+
+                            </div>
+                        </div>
+
+                        <div class="action-sec">
+                            <button>
+                                <span>See Project</span>
+                            </button>
+                        </div>
+
+                        <div class="mask-group">
+                            <div class="right-bloc">
+                                <h2 class="title-post">
+                                    Professional Art Printing Data
+                                </h2>
+                                <p class="other-cards-supporting-text">A daily selection of privately personalized
+                                    reads; no accounts or sign-ups required.
+                                    has been the industry's standard
+                                </p>
+                                <ul class="other-tags">
+                                    <li class="html"><span>html</span></li>
+                                    <li class="bootstrap"><span>bootstrap</span></li>
+                                    <li class="ruby"><span>Ruby</span></li>
+                                </ul>
+
+                            </div>
+                        </div>
+
+                        <div class="action-sec">
+                            <button>
+                                <span>See Project</span>
+                            </button>
+                        </div>
+
+                        <div class="mask-group">
+                            <div class="right-bloc">
+                                <h2 class="title-post">
+                                    Professional Art Printing Data
+                                </h2>
+                                <p class="other-cards-supporting-text">A daily selection of privately personalized
+                                    reads; no accounts or sign-ups required.
+                                    has been the industry's standard
+                                </p>
+                                <ul class="other-tags">
+                                    <li class="html"><span>html</span></li>
+                                    <li class="bootstrap"><span>bootstrap</span></li>
+                                    <li class="ruby"><span>Ruby</span></li>
+                                </ul>
+
+                            </div>
+                        </div>
+
+                        <div class="action-sec">
+                            <button>
+                                <span>See Project</span>
+                            </button>
+                        </div>
+                    </div>
+
+
+
+
+                </section>
```

Comment:
- [ ] KIndly fix the blank line space in your HTML file for the code to be clean, professional, and [good HTML practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Fixed.